### ISSUE TITLE
Versions Megahit after 1.1.4 do not contain megahit_toolkit?

### ISSUE THREAD

**Issue by dutchscientist** [voutcn/megahit#226]
Megahit is one of the dependencies for Shovill (Torsten Seemann, https://github.com/tseemann/shovill/), and with the new version of Megahit there is no "megahit_toolkit" anymore. This is a dependency that Shovill looks for when installed via Conda, and hence stops running.

I can find version 1.1.1 on Conda, but that only runs in Python 2.7 to 3.6, while Conda3 is now already at Python 3.7.

Is this by design? Issue at Shovill can be seen at:
https://github.com/tseemann/shovill/issues/112

Would be great if this can be resolved, thanks :)

**Comment by voutcn** [comment]
It is removed after 1.1.4. Some useful sub-commands (including `contig2fastg`, the one used by Shovill) have been moved to `megahit_core`. To solve this, I will create an alias of `megahit_core` as `megahit_toolkit` in the next release (in one or two days).

**Comment by dutchscientist** [comment]
Thank you, much appreciated! :)

**Comment by tseemann** [comment]
Thank you @voutcn and @dutchscientist for fixing this and maintaining backward compatibility!

**Comment by JHarrisonEcoEvo** [comment]
It seems like there is still some problems with legacy megahit_toolkit commands not working. 

I just tried the following command to convert contigs to fastg: 

megahit_core contig2fastq 99 ../intermediate_contigs/k99.contigs.fa > k99.contigs.fastg

I also tried the previous accepted command of:

megahit_toolkit contig2fastq 99 ../intermediate_contigs/k99.contigs.fa > k99.contigs.fastg

Both provide the usage help to stdout but do not throw any errors or write anything to the output file. There is nothing germane in the usage prompt ( I copied it below for reference).  My input contigs look like this (just first two contigs): 

>k99_0 flag=1 multi=3.0000 len=526
ACCCTGCTTTCCGAAGCGGGCGACACGGCCGCCGCGATCTCGGCCTTCTCGTCGATCGGCAAGGACCGCTCCGTTCCGGAAGCGATCCGCGATGCGGCCAAGCTGCGCGCCGGCTGGCTGCTGGTCGATACCGGCACCTATGAGCAGGTTTCCGCCGAGGTCGAGGAACTGTCGAATGACGGCAATCCGTTCCGCTTCTCCGCCCGCGAAATACTCGGACTGTCGGCCTATCGGCTGAAGGATTACACCCGCGCCCAGCAATGGTTCGAGGCGATTGCCAACGACCCGGAAGCCCCGCGCAACGTGCTCAACCGCGCCCAGATCATGCTCGAGGTGATGACCGCCGACGGCAATGTCGCCGAAAGCGCGGCCGGCCGATCAAGCTGTTCGACACCGCCGGCATGCGCCGCAAGGCGCGGGTGGTCGAAAAGCTGGAAAAGCTCTCGGTTTCCGACGCTCTGCGTGCGATCCGGTTTGCCGAATGCGTGGTGATCGTCTTCGATGCGACGATCCCGTTTGAAAAACA
>k99_4539 flag=1 multi=3.0000 len=798
CAGCGCCTCGCCATTGTCTCCCACCAGGTCGGGGGCGAGCGCGCGAATGTCATAGGTCAGATCGACATCCTCGGAGAACCGCCGGATGACGCCATAGGCCTTCGACAGCGAAGTGCCGCCCTTGAATACCAGATGTTCGCCGAGCAGCGAGCCGTAGAGCGTGGCCAGCGCCCACACCACCCACGCATCCTTCTCCAGCAGGTGCGCGGGTCGGCCCGAGCGGTCGGCGGCAACGCTCAATGCTTCAAGGCGATCCTCGGCCGGAAGGAGGAGGAAGGGGTCAGCCATGCGCGACCTTACCCACGCTGCGCGCGAGCCAGGTCGGAAGCTGAGGCGCCGCCGCCACCAGCTCGCTGAAAGCGGCAGGCGGAAGCTTGCGCTTCAGGGTCTGGAGCGCCGACTCAGCCCTCTCCGGTCCGAGCCAAGCAAGCGCCCGCACCGCCTGTCCCGCCGGCCGATGCGCCATGGCAAGTTGCCAGCGCGGAGCGTGACGAAGCTCGACGGTCTGCTTGCCTAGGCTCATCTTCCGGGTGCGTCCCGAGGTCAGATAGACCGACCGGACCGGCACCTGCGTCGTAAGGCCGAGGCTGTTGGCGGCCGCCGCACCGCTTGAGACGATGACCTCGCCGCGCTGGGCTGCAAGCGCCTCAACCGCCTGCTCCACCGATGGCGCCCGGGTGCCGAACCGGCTCGTGACCGGCAGCAGATAAACACCTCGCCCGGCCCGAAGCAGTTGGCCCCGCTCGGCCAGACGCGACAAGGCCTGGTCCACCGCGGCGCGATTGCCAAGGTGGAGCA

What is printed when I run the above referenced calls to megahit_core or toolkit

Usage: megahit_core <sub_program> [sub options]
    sub-programs:
       assemble       assemble from SdBG
       local          local asssembly
       iterate        extract iterative edges
       buildlib       build read library
       count          kmer counting
       read2sdbg      build sdbg from reads
       seq2sdbg       build sdbg from megahit contigs + edges
       contig2fastg   convert MEGAHIT's k*.contigs.fa to fastg format
       readstat       calculate read stats (# of reads, bases, longest, shortest, average)
       filterbylen    filter contigs by length
       checkcpu       check whether the run-time CPU supports POPCNT and BMI2
       checkpopcnt    check whether the run-time CPU supports POPCNT
       checkbmi2      check whether the run-time CPU supports BMI2
       dumpversion    dump version
       kmax           the largest k value supported
